tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/static_method_tests.bal

```diff
@@ -36,6 +36,21 @@ public type ApiDefinition record {|
 
 public type MyType byte[]|string|int|float;
 
+function testJavaNullPointerException() {
+    error? unionResult = trap callThrowNPEWithCallback();
+    test:assertEquals(unionResult is error, true);
+    if unionResult is error {
+        test:assertEquals(unionResult.message(), "java.lang.NullPointerException");
```

Comment:
Shall we assert the stacktrace as well.

Reply:
Previously stack trace had not been added to BError. Hence stack trace was only available in crash logs but not in the panicked BError. So, added the stacktrace to that too with fcc011b599830e48e5bbfb0eecf5e9e9da46970b. Now the output when panicked is like
```
Running executable

[2022-09-23 13:50:48,317] SEVERE {b7a.log.crash} - null 
java.lang.NullPointerException
        at SimpleInterop.throwNPE(SimpleInterop.java:3)
        at nadeeshand.scheduler_hang.0.main.throwNPE(main.bal:33)
        at nadeeshand.scheduler_hang.0.main.$lambda$_0(main.bal:29)
        at nadeeshand.scheduler_hang.0.main.$lambda$_0$lambda0$(main.bal:28)
        at io.ballerina.runtime.internal.scheduling.SchedulerItem.execute(Scheduler.java:594)
        at io.ballerina.runtime.internal.scheduling.Scheduler.run(Scheduler.java:321)
        at io.ballerina.runtime.internal.scheduling.Scheduler.runSafely(Scheduler.java:288)
        at java.base/java.lang.Thread.run(Thread.java:829)
 
error: java.lang.NullPointerException
        at nadeeshand.scheduler_hang.0:throwNPE(main.bal:33)
           nadeeshand.scheduler_hang.0:$lambda$_0(main.bal:29)
```